benches/oneshot/ipa.rs

```diff
@@ -18,6 +18,10 @@ use std::{num::NonZeroUsize, time::Instant};
 #[global_allocator]
 static GLOBAL: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
```

Comment:
You should disable the jemalloc allocator above when you enable heap profiling.  Or ensure that the two cannot be simultaneously enabled.